cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -28,26 +28,280 @@ Cask Data Application Platform Release Notes
 
 New Features
 ------------
-- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+- `CDAP-20 <https://issues.cask.co/browse/CDAP-20>`__ - Removed dependency on the Guava
```

Comment:
If that is the case...yes...and maybe we should give an example of how (ie, what in the pom.xml it needs?)